src/main/kotlin/com/coder/gateway/sdk/CoderRestClientService.kt

```diff
@@ -218,3 +248,168 @@ class CoderRestClient(var url: URL, var token: String,
         }
     }
 }
+
+fun coderSocketFactory() : SSLSocketFactory {
+    val state: CoderSettingsState = service()
```

Comment:
Looks like this usage is causing issues for the tests, not sure what the "correct" way to make settings available to this code ....

Reply:
I updated the code to have the settings passed in.